backtracking/combination_sum_II.py

```diff
@@ -0,0 +1,54 @@
+"""
+Time complexity(Average Case): 0(m)
+
+Constraints:
+The input n is a positive integer.
+The input n is bounded by 1 <= n <= 30.
+This constraint is typically imposed to limit the size of the output string,
+as the "Count and Say" sequence grows exponentially with n.
+"""
+
+
+class Solution:
```

Comment:
A class named `Solution` is almost always a bad idea especially when it has no variables.  You could lose the `class` and just have a function called `count_and_say()` but that function name has nothing to do with `combination_sum` which is the filename.  If we already have a `combination_sum` then I doubt we need a second one.  Closing...  We have more than 100 open pull requests.